docs/README.md

```diff
@@ -128,6 +128,36 @@ cleanlab-docs
 
 7. Navigate to the URL where your GitHub Pages site is published in step 3. The default URL should have the format *https://repository_owner.github.io/cleanlab-docs/*.
 
+# Manually adding build artifacts to the `cleanlab-docs` repo
+
+GitHub Actions automatically builds and deploys the docs' build artifacts when [triggered](#cicd-for-cleanlab-docs). If you delete and recreate a release tag, the docs for this tag will be rebuilt and redeployed, hence overwriting the existing artifacts with the new ones.
+
+On rare occasions, you may want to update the docs without deleting and recreating the release tag, for example, when you want to fix a typo in the docs, but you've already deployed your tag to PyPI or Conda. This can be done by manually adding specific docs' build artifacts to the `cleanlab/cleanlab-docs` repo. These steps are for users who have `push` permission to `cleanlab/cleanlab` and `cleanlab/cleanlab-docs` repo.
+
+1. If you haven't already done so, [clone](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository) the `cleanlab/cleanlab` repo.
```

Comment:
Is there a particular reason your link for `clone` is for cloning a forked repo vs say this official link:
https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository

The text does not mention working on a fork anywhere.

Reply:
my bad, it's supposed to be on the upstream `cleanlab/cleanlab` repo. not a forked one. fixed this in the latest push